### DISCUSSION TITLE
Option to render only the latest GeoJSON message in a Map panel

### DISCUSSION THREAD

**Discussion by jemeador** [foxglove/community#790] [Q&A]
When using a UserScript to output GeoJSON based on another topic in a pre-recorded MCAP file, it appears that every message adds to the Map instead of rendering just the latest message. 

This can cause severe performance problems if there are a lot of messages. Consider the use case of rendering the bounding boxes of multiple moving vehicles with GeoJSON.

**Comment by foxymiles** [comment]
We could mitigate this by adding a setting to control whether we show all GeoJSON messages or just the latest one.
